### PR TITLE
Allow to add annotations to router service in helm chart

### LINKED ISSUE #1337: Not able to configure fission router to use internal loadbalancer
While installing fission using helm chart in kubernetes cluster, K8s service for fission router is created as external loadbalancer exposing it outside

There is no way to configure the helm chart to make it the fission router to be exposed via internal loadbalancer

**Expected result**
Allow to add annotations to fission router in helm chart, so that it can use internal loadbalancer

### PR DESCRIPTION
**Pull request by prabhu43**
Fixes #1337

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1338)

### CODE REVIEW

File: charts/fission-all/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fission-all
-version: 1.5.0
+version: 1.6.0
 description: Fission is a fast serverless framework for Kubernetes.
```
Comment (life1347) [inline charts/fission-all/Chart.yaml]: We will change this when creating a new release, please rollback the changes. (fission-all and fission-core)

File: charts/fission-all/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fission-all
-version: 1.5.0
```
Comment (vishal-biyani) [inline charts/fission-all/Chart.yaml]: Please don't change the chart version as part of this PR - that will be changed as part of Fission release 1.6

File: charts/fission-core/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fission-core
-version: 1.5.0
```
Comment (vishal-biyani) [inline charts/fission-core/Chart.yaml]: Please don't change the chart version as part of this PR - that will be changed as part of Fission release 1.6

**Comment by vishal-biyani** [comment]
@prabhu43 - thanks for this useful PR. Please address two small issues I have commented on.

**Comment by prabhu43** [comment]
@vishal-biyani @life1347 Thanks for your feedback. I changed the chart version back to 1.5.0 and updated the chart readme file. Also I moved this config to `router.svcAnnotations`

**Comment by life1347** [comment]
LGTM

**Pull request merged**